e2e/test/DeviceTokenRefreshE2ETests.cs

```diff
@@ -157,10 +156,7 @@ public int SafeCreateNewTokenCallCount
             {
                 get
                 {
-                    lock (_lock)
-                    {
-                        return _callCount;
-                    }
+                    return _callCount;
```

Comment:
Why not interlocked? These can / will be on different threads.

Reply:
done